docs/guides/download_output.md

```diff
@@ -0,0 +1,28 @@
+!!! important "Quick Look"
+    On the job screen on floydhub.com:
+
+    ![Download Output](../img/download_output_bar_only.jpg)
+
+    Or from the CLI:
+    ```
+    $ floyd output username/project_name/run_number
```

Comment:
The equivalent of download button from the web in cli is `floyd data clone username/project_name/run_number/output`. We can also add it to the list.

Reply:
I'll clarify that

Reply:
Okay I just split them up. It's a little clearer that way